projects/RabbitMQ.Client/Impl/ChannelBase.cs

```diff
@@ -1938,17 +1879,9 @@ void MaybeAddPublishSequenceNumberToHeaders(IDictionary<string, object?> headers
             {
                 if (_publisherConfirmationsEnabled)
                 {
-                    byte[] publishSequenceNumberBytes;
-                    if (BitConverter.IsLittleEndian)
-                    {
-                        publishSequenceNumberBytes = BitConverter.GetBytes(BinaryPrimitives.ReverseEndianness(publishSequenceNumber));
-                    }
-                    else
-                    {
-                        publishSequenceNumberBytes = BitConverter.GetBytes(publishSequenceNumber);
-                    }
-
-                    headers[Constants.PublishSequenceNumberHeader] = publishSequenceNumberBytes;
+                    Span<byte> publishSequenceNumberBytes = stackalloc byte[8];
```

Comment:
There is probably not point in stackalloc and then right after that doing a ToArray. What was the intend of this change?

Reply:
Well I was wondering about that. I can revert it!